core/src/test/java/org/apache/calcite/test/JdbcTest.java

```diff
@@ -3504,6 +3504,238 @@ public void checkOrderBy(final boolean desc,
             "empid=110; name=Theodore");
   }
 
+  /** Test case for
```

Comment:
This test is too long. Please put the test into `ReflectiveSchemaTest` and simplify it. You can refer to `ReflectiveSchemaTest.java#testOp`.